torch/csrc/xpu/Module.cpp

```diff
@@ -247,14 +250,18 @@ static void registerXpuDeviceProperties(PyObject* module) {
       .def_readonly("has_fp64", &DeviceProp::has_fp64)
       .def_readonly("has_atomic64", &DeviceProp::has_atomic64)
       .def_property_readonly("type", get_device_type)
+      .def_property_readonly("device_arch", get_device_architecture)
```

Comment:
```suggestion
      .def_readonly("architecture", &DeviceProp::architecture)
```

Reply:
I'm not sure how to cast from `architecture` to `int` using this way. The only option documentation suggests in this case is to import whole enum https://pybind11.readthedocs.io/en/stable/classes.html#enumerations-and-internal-types , but considering other comment it is not the intension.